scaffold/cmd/templates/source/Makefile.tpl

```diff
@@ -0,0 +1,16 @@
+.PHONY: test
+test:
+	go test -timeout 3m ./...
+
+.PHONY: lint
+lint:
+	@golangci-lint run --timeout 10m --verbose
```

Comment:
```suggestion
	@golangci-lint run --timeout 10m
```

Non blocking, I find that `--verbose` makes it very hard to find the actual errors reported by the linter